frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2077,6 +2078,27 @@ Component that allows to receive contributions from other plugins for the `conso
 
 
 
+---
+
+## `NamespaceBar`
+
+### Summary 
+
+A component that renders a namespace dropdown menu.
```

Comment:
We need more detail. Based on the description, I would assume it's a dropdown I could use in a form for instance.

We should describe what the namespace bar is, where it appears, and when/why you would want to use it. I would also add a code example. Assume the person reading this doesn't know anything about the console code base and isn't as familiar with the UI as we are working on this day to day.

Reply:
I updated it. Let me know if you don't think it's descriptive enough.